### PR TITLE
vz-4328 update release notes for 1.1

### PR DESCRIPTION
**Pull request by mgianatagh**
Updated release notes for 1.1

### CODE REVIEW

**Review by markxnelson — APPROVED** [review]

File: content/en/docs/releasenotes/_index.md [hunk 1]
```diff
@@ -7,8 +7,19 @@ draft: false
 
 ### v1.1.0
 Fixes:
+- Added support for Kiali
+- Simplified the placement of multicluster resources.
+- Improved the performance of installing Verrazzano
+- Added support for for external ElasticSearch
```
Comment (bhatpmk) [inline content/en/docs/releasenotes/_index.md:13]: There is an extra "for"

Comment (mgianatagh) [reply content/en/docs/releasenotes/_index.md]: I fixed the typo and changed to Elasticsearch

File: content/en/docs/releasenotes/_index.md [hunk 2]
```diff
@@ -7,8 +7,19 @@ draft: false
 
 ### v1.1.0
 Fixes:
+- Added support for Kiali
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:10]: Please put a period at the end of each bulleted item.

File: content/en/docs/releasenotes/_index.md [hunk 3]
```diff
@@ -7,8 +7,19 @@ draft: false
...
+- Added support for external Elasticsearch
+- Improvements to system functions, including the authenticating proxy
+- Added support in the LoggingTrait to customize application logging
+- Fixed ability to register a managed cluster with Rancher when configured to use LetsEncrypt staging certificates.
+- Fixed Elasticsearch status yellow due to unassigned shards
+- Added support for Kubernetes 1.21, dropped support of Kubernetes 1.18
 - Updated several installed and supported [Software Versions]({{< relref "/docs/setup/versions.md" >}}).
 
+Known Issues:
+- Importing a Kubernetes v1.21 cluster into Rancher might not work properly. Rancher is planning to add support for Kubernetes v1.21 in the future.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:22]: "Rancher is planning to add support for Kubernetes v1.21 in the future." This statement is a No-No. We cannot state what Rancher is planning to do. Period. You can say, "Rancher does not currently support Kubernetes v1.21, or whatever else is the technically correct reason why "Importing a Kubernetes v1.21 cluster into Rancher might not work properly."

**Review by rosemarymarano — COMMENTED** [review]
few edits

**Review by rosemarymarano — APPROVED** [review]

**Review by gilbode — APPROVED** [review]

**Pull request merged**